assemblies/assembly_openshift-authoring-managed/main.adoc

```diff
@@ -39,7 +39,8 @@ include::{enterprise-dir}/openshift/imagestreams-file-install-proc.adoc[leveloff
 // include::{enterprise-dir}/openshift/registry-auth-proc.adoc[leveloffset=+2]
 include::{enterprise-dir}/openshift/secrets-kie-create-proc.adoc[leveloffset=+2]
 include::{enterprise-dir}/openshift/secrets-central-create-proc.adoc[leveloffset=+2]
-include::{enterprise-dir}/openshift/glusterfs-reconfig-proc.adoc[leveloffset=+2]
+include::{enterprise-dir}/openshift/offline-repo-proc.adoc[leveloffset=+2]
+// include::{enterprise-dir}/openshift/glusterfs-reconfig-proc.adoc[leveloffset=+2]
```

Comment:
Why is this line commented out?

Reply:
It is commented out because this is a DM-only document and GlusterFS is to be reconfigured for PAM. I missed this previously and found it now, so just fixed along the way.

Reply:
What is the difference between DM and PAM related to usage of GlusterFS? AFAIK both need it.